app/screens/products/styles.js

```diff
@@ -1,7 +1,7 @@
-import { StyleSheet } from 'react-native';
+import {StyleSheet} from 'react-native';
 
-export default StyleSheet.create({
-  container: {
-  
-  }
-});
\ No newline at end of file
+export default StyleSheet.create ({
+  container: {flex: 1, padding: 20, paddingTop: 20, backgroundColor: '#fff'},
```

Comment:
for readability purposes, after a parameter always move to the next line.

e.g. 
container: {
flex:1, 
padding: 20,
...
}